### ISSUE TITLE
Printer friendly device list

### ISSUE THREAD

**Issue by JsChiSurf** [bwssytems/ha-bridge#583]
My wife has been "nagging" me for quite some time now to print her a list of device names, as she says she can never remember what to call.  I went ahead and saved a backup via the UI and used the backup file to parse the JSON in PHP and output a nice user friendly list.

However, it would be great, and "wife factor" approved if a button could be integrated into the UI that would allow us to print a nice printer friendly list for those in the house who don't set this stuff up, and thus aren't as familiar with how things are named.

**Comment by roybrew** [comment]
I would agree!  I wondered that myself.  Even if a CSV list could be dumped out/exported you could pull that into a spreadsheet.

**Comment by bwssytems** [comment]
The button will be labeled "WAF"....

**Comment by bwssytems** [comment]
Anyone still interested in this enhancement?

**Comment by roybrew** [comment]
Would be a "nice to have"

**Comment by Goggo16** [comment]
Yep, would also be interested.

My set of Alexa-Devices within HA-Bridge keeps growing. And sometimes, I don't remember the exact wording to some of those. ;)

Regards, Goggo

PS: This is my first post in Github. So please forgive me, if I may have missed some etituette. And BTW, Habridge is great.

**Comment by JsChiSurf** [comment]
Yep, still would love to see this as an option.
